modules/1_bastion/templates/create_disk_link.sh

```diff
@@ -1,14 +1,23 @@
-#Scan devices
 sudo rescan-scsi-bus.sh -a -m -r
 
 storage_device=""
 storage_disk_name=${disk_name}
-for device in $(ls -1 /dev/mapper/mpath*|egrep -v "[0-9]$"); do
+
+if [[ -z $(ls -l /dev/mapper/mpath*) ]];then
+    disk_path=/dev/sd*
+    echo "Disk path is /dev/sd*"
+else
+    disk_path=/dev/mapper/mpath*
+    echo "Disk path is /dev/mapper/mpath*"
+fi
+
+for device in $(ls -1 $disk_path|egrep -v "[0-9]$"); do
     if [[ ! -b $device"1" ]]; then
         # Convert disk size to GB
         device_size=$(lsblk -b -dn -o SIZE $device | awk '{print $1/1073741824}')
         if [[ -z $storage_device && $device_size == ${volume_size} ]]; then
             storage_device=$device
+            echo "NFS disk is $device"
```

Comment:
One minor comment. Let's change "NFS"-> "Storage".

Reply:
updated and pushed the commit